src/pretix/static/pretixcontrol/js/ui/editor.js

```diff
@@ -397,6 +398,7 @@ var editor = {
         editor.fabric.on('object:added', editor._create_savepoint);
         editor.fabric.on('selection:cleared', editor._update_toolbox);
         editor.fabric.on('selection:created', editor._update_toolbox);
+        editor.fabric.on('selection:updated', editor._update_toolbox);
         editor.fabric.on('object:selected', editor._update_toolbox);
         editor.fabric.on('object:moving', editor._update_toolbox_values);
```

Comment:
As noted in the [changelog](http://fabricjs.com/v5-breaking-changes) for v5, `object:moving, object:rotating` and `object:scaling` are just an „extra layer detail on top of `object:modified`“. As all of these events call `editor._update_toolbox` all `editor.fabric.on('object:XXX',` except `editor.fabric.on('object:modified',` could/should be removed. See lines 405 and 406 as well – could not select them in the UI here.

```suggestion
```